help/destinations/catalog/email-marketing/mailchimp-tags.md

```diff
@@ -0,0 +1,173 @@
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+>
+>To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]**, search for [!DNL Mailchimp Tags]. Alternatively you can locate it under the **[!UICONTROL Email marketing]** category.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**. You will be navigated to the [!DNL Mailchimp Log in and authorize] page. Enter your [!DNL Mailchimp] account credentials and select [!DNL Log In].
+
+![Platform UI screenshot showing how to authenticate to Mailchimp.](../../assets/catalog/email-marketing/mailchimp-tags/authenticate-destination.png)
+
+Next, Select [!UICONTROL Allow] in the subsequent window to give permissions to the **Adobe Experience Platform** app to access your [!DNL Mailchimp] account. *You will need to do this only once*.
+
+![Mailchimp App screenshot confirmation popup to give permissions to the Experience Platform app access to Mailchimp.](../../assets/catalog/email-marketing/mailchimp-tags/allow-app.png)
+
+If the details provided are valid, the UI displays a message: *You successfully connected to Mailchimp Tags account* message and a **[!UICONTROL Connected]** status with a green check mark, you can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Platform UI screenshot showing the destination details.](../../assets/catalog/email-marketing/mailchimp-tags/destination-details.png)
+
+| Field | Description |
+| --- | --- |
+| **[!UICONTROL Name]** | A name by which you will recognize this destination in the future. |
+| **[!UICONTROL Description]** | A description that will help you identify this destination in the future. |
+| **[!UICONTROL Data Center]** | Your [!DNL Mailchimp] account `Data Center`. Refer to the [Identify [!DNL Mailchimp] data center](#identify-data-center) section for any guidance. |
+| **[!UICONTROL Audience List ID]** | After you provide the value of your `Data Center` entries in this select dropdown are automatically populated with the audience names from your account. You need to select the audience of interest to which you will add or update the contact details from selected Platform segments. |
```

Comment:
After you enter your [!UICONTROL Data Center] prefix, this dropdown is automatically populated with the audience names from your account. Select the audience that you want to update with data from Platform.